src/pages/auth/components/RegisterForm.jsx

```diff
@@ -78,12 +75,14 @@ export const RegisterForm = () => {
             isError={errors.password && touched.password}
           />
           <Button
-            style={{ marginTop: '8px' }}
+            style={{ marginTop: '15px' }}
+            buttonTextProps={{ lineHeight: 24, fontSize: 18 }}
             type='submit'
             fullWidth
-            rightIcon={<LoginIcon />}
-            title={t('Sign Up')}
+            rightIcon={<LoginIcon color={'white'} />}
+            title={isLoading ? t('Loading') : t('Sign up')}
```

Comment:
the same, you've already put isLoading prop, why you don't use this logic inside for every buttons?

Reply:
done